config/zones/US-SW-EPE.yaml

```diff
@@ -172,74 +174,74 @@ fallbackZoneMixes:
       datetime: '2017-01-01'
       value:
         battery discharge: 0
-        biomass: 0.000003853100112863885
+        biomass: 3.853100112863885e-06
```

Comment:
No idea why the formatting changed here

Reply:
Prettier or the Yaml package, It should be fine either way.